### ISSUE TITLE
Issue Saving Edited Task

### ISSUE THREAD

**Issue by AllenKennedy** [CIRDLES/Squid#692]
If I make a copy of a Task, including Custom Eqns and then try to save it, a different title appears in the saving window
[Allanite C36 A46 3rd try.squid.zip](https://github.com/CIRDLES/Squid/files/8348665/Allanite.C36.A46.3rd.try.squid.zip)
.

**Comment by NicoleRayner** [comment]
I'm not sure if this is the same issue you are having, but I found something similar and it was because "TASK NAME" does not automactically change if it is saved as an xml with a new name.
![image](https://user-images.githubusercontent.com/40035242/160126838-c9edaa5b-06f0-4b95-8aa1-330488543a7c.png)

Thus if you edited an old task and renamed **the xml,** but didn't rename it in the spot above, it would show you the original name.
Granted I did not look at your attached squid file so I am not sure if this is really the problem you are having, but it was one that I had.
